express/parsers/apps/nwchem/formats/txt.py

```diff
@@ -0,0 +1,51 @@
+import os
+import io
+import re
+import numpy as np
+
+from express.parsers.utils import find_file
+from express.parsers.settings import Constant
+from express.parsers.apps.nwchem import settings
+from express.parsers.formats.txt import BaseTXTParser
+
+
+class NwchemTXTParser(BaseTXTParser):
+    """
+    Nwchem text parser class.
+    """
+
+    def __init__(self, work_dir):
+        super(NwchemTXTParser, self).__init__(work_dir)
+
+    def total_energy(self, text):
+        """
+        Extracts total energy.
+
+        Args:
+            text (str): text to extract data from.
+
+        Returns:
+             float
+        """
+        return self._general_output_parser(text, **settings.REGEX["total_energy"])
+
+    # NEED TO CHECK MATH/UNITS ON TOTAL ENERGY CONTRIBUTION
```

Comment:
Let's do the checks and remove the comment, pls

Reply:
yes I will take care of that.